tests/page-objects/reports/reports.po.js

```diff
@@ -136,5 +209,12 @@ module.exports = {
       'There should be at least one report in the LHS'
     );
   },
+
+  waitForReportToAppearNative: async (numOfReports = 1) => {
+    await browser.refresh();
+    while (await module.exports.allReports().count < numOfReports) {
+      browser.sleep(100);
```

Comment:
Without the `await`, this line won't have any "sleep" impact.
```suggestion
      await browser.sleep(100);
```